meegkit/trca.py

```diff
@@ -126,7 +139,9 @@ def fit(self, X, y):
             eeg_tmp = X[..., y == class_i]
             for fb_i in range(self.n_bands):
                 # Filter the signal with fb_i
-                eeg_tmp = filterbank(eeg_tmp, self.fs, fb_i)
+                eeg_tmp = bandpass(eeg_tmp, self.sfreq,
+                                   Wp=self.filterbank[fb_i][0],
```

Comment:
Following the modification for filterbank...

Wp=[self.filterbank[fb_i][0], self.filterbank[fb_i][1]]
Ws=[self.filterbank[fb_i][2], self.filterbank[fb_i][3]]